woocommerce/woocommerce-android
# [Login Prologue] Add new to woocommerce button

Closes: #6943 


### Description
This PR adds a button "new to woocommerce" in the login prologue, to allow new users to learn more about Woo as a platform. 

### Testing instructions
1. Launch the app (make sure you are signed out).
2. Check the design in different orientations/screen sizes.
3. Click on the button.
4. Confirm it launches a webview pointing to our docs.
5. Confirm `Tracked: login_new_to_woo_button_tapped` is printed to the logcat.

### Images/gif
| | | |
| - | - | - |
| Phone | ![image](https://user-images.githubusercontent.com/1657201/178974745-238249d0-861c-4b43-99a4-6045a02eddde.png) | ![image](https://user-images.githubusercontent.com/1657201/178974805-c26c0781-c3ed-4648-9209-fac7029ac1fe.png) |
| Tablet | ![image](https://user-images.githubusercontent.com/1657201/178974897-93d1b244-98c8-4b3a-a9e6-5201cea6c97d.png) | ![image](https://user-images.githubusercontent.com/1657201/178974849-b2fc1c19-cb3b-40d5-8325-49f77ce5ddd6.png) |
